uniswap/uniswap.py

```diff
@@ -78,6 +79,7 @@ def __init__(
         # use_eip1559: bool = True,
         factory_contract_addr: str = None,
         router_contract_addr: str = None,
+        enable_caching: bool = False,
```

Comment:
Is this not safe to set to `True` now, given the whitelist?

Reply:
Should be safe, unless middleware adds some unintended behaviour. Haven't read that much about them. For all current users don't wanted to force them to run with middleware. I rather want them to read about middleware and set this flag manually.

Reply:
Agreed!